src/vscode-bicep/package.json

```diff
@@ -97,7 +97,7 @@
       {
         "language": "bicep",
         "scopeName": "source.bicep",
-        "path": "./bicep.tmlanguage"
+        "path": "./syntaxes/bicep.tmlanguage"
```

Comment:
Seems to be a convention to put grammar files in the `syntaxes` folder:

![image](https://user-images.githubusercontent.com/16367959/118361806-8b934e80-b5bf-11eb-87cd-2037b56b1843.png)